administrator/language/en-GB/com_cookiemanager.ini

```diff
@@ -0,0 +1,11 @@
+; Joomla! Project
+; (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+COM_COOKIEMANAGER="Cookie Manager"
+COM_COOKIEMANAGER_XML_DESCRIPTION="Cookie manager to manage cookies."
```

Comment:
At the end, the language keys mus be in alphabetical order.

Reply:
Ok, at the end, I will sort it in the alphabetical order.